test/Directory.Build.props

```diff
@@ -4,5 +4,6 @@
   <!-- Build configuration -->
   <PropertyGroup>
     <TargetFrameworks>net461;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT' OR '$(CoreOnly)' == 'True'">netcoreapp3.0</TargetFrameworks>
```

Comment:
This should no longer be necessary - we now reference Microsoft.NETFramework.ReferenceAssemblies from Directory.Build.props at the root, so Linux and Mac should compile for .NET Framework just fine. Running tests is a different matter :) That would be the mono testing...

Reply:
Yeah, mono was throwing, so I borrowed this from the EFCore.PG test setup. Should this be done in a different way?

Reply:
It would be somewhat better to simply not run the tests on mono in devops (so configuration via the YAML), but I'm also OK with disabling the tests this way. Just maybe add a comment to explain that it's about actual test failures.

One more option would be to actually run the tests in CI but not fail the build on them, but I wouldn't spend more than 5 minutes making that happen.